jwst/datamodels/container.py

```diff
@@ -8,6 +8,14 @@
 from asdf import AsdfFile
 from astropy.io import fits
 
+from jwst.associations.lib.dms_base import (
+    CORON_EXP_TYPES,
+    IMAGE2_NONSCIENCE_EXP_TYPES,
+    IMAGE2_SCIENCE_EXP_TYPES,
+    SPEC2_SCIENCE_EXP_TYPES,
+    ACQ_EXP_TYPES
+)
```

Comment:
These are all lists of EXP_TYPE values, which are recorded as keyword values in the headers of the member files. They aren't recorded in the ASN file itself. I'm guessing what you really meant is a list of ASN member types, which is recorded in the "exptype" attribute in the ASN json files, right? Those have values like "science", "background", "psf", "target-acquisition", etc., and I'm guessing are stored in some other macros somewhere within associations/lib.